tests/core/config/test_config_serialization.py

```diff
@@ -58,7 +58,7 @@ def default(self, o):
 
 class CustomDecoder(json.JSONDecoder):
     def __init__(self, *args, **kwargs):
-        json.JSONDecoder.__init__(self, object_hook=self.object_hook, *args, **kwargs)
+        json.JSONDecoder.__init__(self, *args, **kwargs, object_hook=self.object_hook)
```

Comment:
... looks strange to me (again)

Reply:
+1

Reply:
The idea is that keyword arguments should be at the end